src/AzureStack/Admin/BackupAdmin/Backup.Admin/Properties/AssemblyInfo.cs

```diff
@@ -15,4 +15,4 @@
 [assembly: NeutralResourcesLanguage("en")]
 
 [assembly: AssemblyVersion("0.1.0.0")]
```

Comment:
For Versions less than 1.0.0, both `AssemblyVersion` and `AssemblyFileVersion`  must be updated

Reply:
Fixed, thanks